examples/Python/SPECT/SPECT_OSEM.py

```diff
@@ -0,0 +1,189 @@
+'''
+Simple OSEM reconstruction demo: creates an image, projects it to simulate
+acquisition data , adds noise to the data and then does an OSEM reconstruction.
+
+Usage:
+  SPECT_OSEM [--help | options]
+
+Options:
+  -f <file>, --file=<file>    raw data file [default: template_sinogram.hs]
+  -p <path>, --path=<path>    path to data files, defaults to data/examples/SPECT
+                              subfolder of SIRF root folder
+  -o <file>, --output=<file>  output file for simulated data
+
+There is an interactive demo with much more documentation on this process.
+You probably want to check that instead.
+'''
+
+## CCP PETMR Synergistic Image Reconstruction Framework (SIRF)
+## Copyright 2015 - 2017 Rutherford Appleton Laboratory STFC
+## Copyright 2015 - 2019 University College London.
+##
+## This is software developed for the Collaborative Computational
+## Project in Synergistic Reconstruction for Biomedical Imaging (SyneRBI)
+## (http://www.ccppetmr.ac.uk/).
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+##   you may not use this file except in compliance with the License.
+##   You may obtain a copy of the License at
+##       http://www.apache.org/licenses/LICENSE-2.0
+##   Unless required by applicable law or agreed to in writing, software
+##   distributed under the License is distributed on an "AS IS" BASIS,
+##   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+##   See the License for the specific language governing permissions and
+##   limitations under the License.
+
+__version__ = '0.1.0'
+from docopt import docopt
+args = docopt(__doc__, version=__version__)
+
+from sirf.Utilities import show_2D_array
+
+# import engine module
+import sirf.STIR
+import numpy as np
+
+# process command-line options
+data_file = args['--file']
+data_path = args['--path']
+if data_path is None:
+    data_path = sirf.STIR.examples_data_path('SPECT')
+raw_data_file = sirf.STIR.existing_filepath(data_path, data_file)
+output_file = args['--output']
+
+def create_sample_image(image, attenuation = False):
+    '''fill the image with some simple geometric shapes'''
+
+    # make shape uniform density for attenuation image
+    if attenuation == True:
+        att = 0
```

Comment:
doesn't make sense to add shapes with `att=0` as far as I can see (just zero image). I'm a bit confused about this

Reply:
scaling factors are raised to the power zero for attenuation image so that they are all scaled the same (1)

Reply:
I've gotten rid of this. It didn't work anyway. image density is now just scaled down for the attenuation image (otherwise the sinogram looks very wrong)